SQL/TSQL.sublime-syntax

```diff
@@ -0,0 +1,231 @@
+%YAML 1.2
+---
+name: T-SQL
+scope: source.sql.tsql
+version: 2
+extends: Packages/SQL/SQL (basic).sublime-syntax
+
+variables:
+  string_escape: (?:'') #(?:\\[tnr])
+contexts:
+  identifier:
+    - meta_prepend: true
+    - include: square-bracketed-identifier
+
+  square-bracketed-identifier:
+    - match: (\[)([^]]+)(\])
+      captures:
+        1: punctuation.definition.identifier.begin.sql
+        3: punctuation.definition.identifier.end.sql
+
+  strings:
+    - meta_append: true
+    - match: N'
+      scope: punctuation.definition.string.begin.sql
+      push: single-quoted-string
+
+  operators:
+    - meta_append: true
+    - match: (?i:\blike\b)
+      scope: keyword.operator.logical.sql
+      branch_point: like-strings-branch
+      branch:
+        - like-string-not-followed-by-escape
+        - like-string-followed-by-escape-slash
+        - like-string-followed-by-escape-caret
+        - like-string-followed-by-unknown-escape
+
+  variables:
+    - match: (@)\w+
+      captures:
+        0: variable.other.readwrite.sql
+        1: punctuation.definition.variable.sql
+    - match: (@@)(?i:rowcount|trancount|error)
```

Comment:
Also `IDENTITY`

Reply:
https://github.com/michaelblyons/SublimeSyntax-Defaults/commit/1baf2de5b40893045301b6fadacfe97c9adc298c#diff-2a1a635547fcbf7145b49d2c363fd7a0c1fa6488732bf7617e868409ff622c2cR12-R13

Reply:
Yep, thanks, lots of known variables and functions still to add :)